bin/cherry-pick.mjs

```diff
@@ -334,6 +334,11 @@ function reportSummaryNextSteps( successes, failures ) {
 		nextSteps.push( 'Push this branch' );
 		nextSteps.push( 'Go to each of the cherry-picked Pull Requests' );
 		nextSteps.push( `Remove the ${ LABEL } label` );
+
+		if ( LABEL === 'Backport to WP Beta/RC' ) {
+			nextSteps.push( 'Add the "Backported to WP Core" label' );
```

Comment:
Should this label be hard coded in the code or should we make it an argument like LABEL?

Reply:
Argument

Reply:
Or at least a const

Reply:
@youknowriad I added a const for now. I don't have a use case for making this another arg yet. If we have one in the future we can easily add this.

It would be useful to bring this change in prior to Beta 3.